package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "plugin-name",
+  "version": "1.0.0",
+  "description": "DESCRIPTION",
+  "main": "index.js",
+  "scripts": {
+    "lint": "tfm-lint --plugin -d /webpack",
+    "test": "tfm-test --plugin",
+    "test:watch": "tfm-test --plugin --watchAll",
+    "test:current": "tfm-test --plugin --watch",
+    "publish-coverage": "tfm-publish-coverage",
+    "stories": "tfm-stories --plugin",
+    "stories:build": "tfm-build-stories --plugin",
```

Comment:
do we also want `stories:deploy` to surge?

Reply:
I think that the storybook deployment should stay consumer's choice

Reply:
true, but we can make it all set for them and they can decide to remove it or not,
I don't think it's too painful to have :)